### PR TITLE
Implemeted FsLoader

### LINKED ISSUE #10: Implement FileSystemLoader

### PR DESCRIPTION
**Pull request by badochov**
closes #10

### CODE REVIEW

File: src/loaders/loaders.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	"github.com/gojinja/gojinja/src/utils/maps"
+	"io/fs"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+func splitTemplatePath(template string) (pieces []string, err error) {
+	for _, piece := range strings.Split(template, "/") {
```
Comment (jmolinski) [inline src/loaders/loaders.go:17]: shouldn't it be split on os.PathSeparator?

File: src/utils/maps/maps.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+func Keys[K comparable, V any](m map[K]V) []K {
+	keys := make([]K, 0, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+	return keys
+}
+
+func OrderedKeys[K constraints.Ordered, V any](m map[K]V) []K {
```
Comment (jmolinski) [inline src/utils/maps/maps.go]: SortedKeys maybe? to be more explicit about what it does

File: src/loaders/loaders.go (hunk 1 above)
Comment (badochov) [reply src/loaders/loaders.go:17]: for some reason jinja uses "/" instead of `os.path.sep`

**Review by jmolinski — APPROVED** [review]
image

**Pull request merged**